xray/commands/testdata/go-example/go.mod.txt

```diff
@@ -0,0 +1,7 @@
+module testGoList
```

Comment:
Let's rename the directory of these files from "go-example" to "go-project", because this is not an example.
Similarly - "pip-example" --> "pip-project".